src/components/atoms/Playground/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { Playground } from '@gnoide/playground'
+import '~/@gnoide/playground/dist/style.css'
+import React from 'react';
+import {
+  useColorMode,
+} from '@chakra-ui/react';
+
+const App = ({ ...props }) => {
+  const files = JSON.parse(decodeURIComponent(props.files)) 
+
+  const { colorMode } = useColorMode();
+  const isLight = colorMode === 'light';
+
+  let theme = {
+    color: "#a8a8a8",
+    primaryColor: '#3a3a3a',
+    secondaryColor: '#1c1c1c',
+    editorBase: 'vs-dark',
+  } 
+
+  if (isLight) {
+    theme = {
+      color: "#606060",
+      primaryColor: '#f1f1f1',
+      secondaryColor: '#bebebe',
+      editorBase: 'vs',
+    } 
```

Comment:
You can probably use the already predefined Chakra theme color constants from `src/theme/theme.ts`, as I can see some of them match

Reply:
done by 61c1541